packages/amazon-sumerian-hosts-babylon/src/Babylon.js/HostObject.js

```diff
@@ -145,6 +155,10 @@ class HostObject extends CoreHostObject {
     // Load character model
     const characterAsset = await BABYLON.SceneLoader.LoadAssetContainerAsync(modelUrl);
     const characterMesh = characterAsset.meshes[0];
+
+    // rename mesh to something human-readable instead of the default '__root__'
+    characterMesh.name = modelUrl.slice(modelUrl.lastIndexOf('/') + 1, modelUrl.lastIndexOf('.'));
```

Comment:
Just checking. You aren't doing any sanitation of the types of characters used for this name value. Is that OK? Does  BabylonJS or the BJS Editor have any specific restrictions around legal name values? For example, is it OK for a name to include ":" or "/" characters, or spaces?

Reply:
The BabylonJS editor simply renders this as a string (sets it [here](https://github.com/BabylonJS/Editor/blob/e3ee4c8f0a695fdf19eeff1ebd76ff5c5dd853bf/src/renderer/editor/components/graph.tsx#L559) for rendering; the property is defined [here](https://github.com/BabylonJS/Babylon.js/blob/3f90b25a6972d88ea92c930e4f3c760b0e16a084/packages/dev/core/src/node.ts#L88) and the constructor for `AbstractMesh` simply passes it up the construction chain without checking/manipulating the string), so there should be no rendering limitations as far as I understand